storybooknative/src/Text/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,791 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Text test1 should match snapshot 1`] = `
+<Text
+  align="center"
+  className="sc-bdVaJa clsgSE"
```

Comment:
why is it rendering className?

Reply:
Maybe because we would need to realise two sets of tests, or rather run them twice, one time for each environment.

Reply:
From https://stackoverflow.com/questions/41482306/how-to-run-jest-test-for-react-native-and-web

>Assuming you using react-native-web to get your HTML output, you should know that Jest won't actually run your react-native code through webpack and thus it won't handle the react-native -> react-native-web alias, so you'll never see the actual HTML in your snapshot.

in which case, it does not really matter that this className is given to us, but it will be of importance for the web.